avm/ptn/app/container-job-toolkit/main.bicep

```diff
@@ -0,0 +1,341 @@
+    tags: tags
+  }
+}
+
+module job 'br/public:avm/res/app/job:0.5.1' = {
+  name: '${uniqueString(deployment().name, location)}-${resourceGroup().name}-appjob'
+  params: {
+    name: '${name}-container-job'
+    location: location
+    enableTelemetry: enableTelemetry
+    tags: tags
+    lock: lock
+    environmentResourceId: services.outputs.managedEnvironmentId
+    workloadProfileName: workloadProfileName
+    managedIdentities: {
+      userAssignedResourceIds: [services.outputs.userManagedIdentityResourceId]
+    }
+    secrets: union(
+      // add Application Insights connection string as secret if provided
+      !empty(services.outputs.keyVaultAppInsightsConnectionStringUrl)
+        ? [
+            {
+              name: 'applicationinsightsconnectionstring'
+              identity: services.outputs.userManagedIdentityResourceId
+              keyVaultUrl: 'https://${services.outputs.vaultName}${environment().suffixes.keyvaultDns}/secrets/applicationinsights-connection-string'
+            }
+          ]
+        : [],
+      // add passed secrets to the job. It uses the Managed Identity to access the secrets.
+      secrets ?? []
+    )
+    triggerType: 'Schedule'
+    scheduleTriggerConfig: {
+      cronExpression: cronExpression
+    }
+    registries: [
+      {
+        identity: services.outputs.userManagedIdentityResourceId
+        server: services.outputs.registryLoginServer
+      }
+    ]
+    containers: [
+      {
+        name: '${name}-scheduled-job'
+        image: import_image.outputs.importedImage.acrHostedImage
+        env: environmentVariables
+        resources: {
+          cpu: cpu
+          memory: memory
+        }
+      }
+    ]
+  }
+}
+
+// ============ //
+// Outputs      //
+// ============ //
+
+@description('The resource ID of the container job.')
+output resourceId string = job.outputs.resourceId
+
+@description('The name of the container job.')
+output name string = job.name
+
+@description('The name of the Resource Group the resource was deployed into.')
+output resourceGroupName string = resourceGroup().name
+
+@description('The resouce ID of the Log Analytics Workspace (passed as parameter value or from the newly created Log Analytics Workspace).')
+output logAnalyticsResourceId string = services.outputs.logAnalyticsResourceId
+
+@description('The principal ID of the user assigned managed identity.')
+output systemAssignedMIPrincipalId string = services.outputs.userManagedIdentityPrincipalId
+
+@description('Conditional. The virtual network resourceId, if a virtual network was deployed. If `deployInVnet` is `false`, this output will be empty.')
+output vnetResourceId string = services.outputs.vnetResourceId
+
+@description('Conditional. The address prefix for the private endpoint subnet, if a virtual network was deployed. If `deployInVnet` is `false`, this output will be empty.')
+output privateEndpointSubnetAddressPrefix string = deployInVnet
```

Comment:
Why the condition here only and not added to other conditional outputs?

Reply:
Uh, probably copy&paste